From 834a137d2f64c52155fb47c13a6fbe1e015c9762 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Wed, 8 Feb 2012 14:03:11 +0800
Subject: ENGR00171642 [MX6Q_Lite]Power:suspend and resume stress test failed

add bus suspend/resume function to prevent SDMMC suspend/resume stess test fail

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 drivers/mmc/core/bus.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/core/bus.c b/drivers/mmc/core/bus.c
index 393d817..56eb056 100644
--- a/drivers/mmc/core/bus.c
+++ b/drivers/mmc/core/bus.c
@@ -167,6 +167,8 @@ static const struct dev_pm_ops mmc_bus_pm_ops = {
 	.runtime_suspend	= mmc_runtime_suspend,
 	.runtime_resume		= mmc_runtime_resume,
 	.runtime_idle		= mmc_runtime_idle,
+	.suspend		= mmc_bus_suspend,
+	.resume			= mmc_bus_resume,
 };
 
 #define MMC_PM_OPS_PTR	(&mmc_bus_pm_ops)
-- 
1.7.9.5

